MessengerOnFileForLinux/MessageFramework/test/MessageTestCore.hpp

```diff
@@ -12,7 +12,12 @@ class MessageFixture : public ::testing::Test
     void TearDown() {};
     ~MessageFixture() = default;
 
-        std::unique_ptr<std::string> expectedReadMessage;
-        std::unique_ptr<std::string> expectedInviterMessage;
-        std::unique_ptr<std::string> expectedRecipientMessage;
+    void comparingExpectedMessageWithCreatedFromFullString(std::string expectedMessage);
+    void comparingExpectedTimeWithTimeFromMessage(std::string expectedMessage);
+    void comparingExpectedUsernameWithUsernameFromMessage(std::string expectedMessage);
+    void comparingExpectedContentWithContentFromMessage(std::string expectedMessage, std::string messageKind);
```

Comment:
messageType brzmi dla mnie lepiej

Reply:
messageType = PurgeMessage / Message
messageKind = read / inviter / recipient

Tak jak to widzę, ale jeśli potrzebujesz to mogę zmienić.

Reply:
ok, to jest to w takim razie mylące, bo po jakbym zobaczył takie dwie nazwy to bym myślałem ze to to samo. Może messageFlag?

Reply:
messageFlag jest zarezerwowane dla enuma i będzie jeszcze bardziej mylące. Chyba, że połączymy to jakoś ze stringiem typu messageFlagString? Choć ja nadal opcjonuję przy messageKind - to jest tylko tutaj w testach.

Reply:
ok